addons/l10n_cn_small_business/data/account.account.template.csv

```diff
@@ -1,76 +1,206 @@
-"l10n_cn_5301","R & D expenditure","5301","account.data_account_type_direct_costs","l10n_cn_small_business.l10n_chart_china_small_business","False"
-"l10n_cn_6001","Main Business Income","6001","account.data_account_type_revenue","l10n_cn_small_business.l10n_chart_china_small_business","False"
-"l10n_cn_6051","Other Business Income","6051","account.data_account_type_other_income","l10n_cn_small_business.l10n_chart_china_small_business","False"
-"l10n_cn_6101","Gains and Losses of fair value change","6101","account.data_account_type_other_income","l10n_cn_small_business.l10n_chart_china_small_business","False"
-"l10n_cn_6111","Income from investment","6111","account.data_account_type_other_income","l10n_cn_small_business.l10n_chart_china_small_business","False"
-"l10n_cn_6301","Non-operating Income","6301","account.data_account_type_other_income","l10n_cn_small_business.l10n_chart_china_small_business","False"
-"l10n_cn_6401","Main Business Cost","6401","account.data_account_type_expenses","l10n_cn_small_business.l10n_chart_china_small_business","False"
-"l10n_cn_6402","Other Operating Costs","6402","account.data_account_type_expenses","l10n_cn_small_business.l10n_chart_china_small_business","False"
-"l10n_cn_6403","Operating Taxes and Surcharges","6403","account.data_account_type_expenses","l10n_cn_small_business.l10n_chart_china_small_business","False"
-"l10n_cn_6601","Selling Expenses","6601","account.data_account_type_expenses","l10n_cn_small_business.l10n_chart_china_small_business","False"
-"l10n_cn_6602","Management Expenses","6602","account.data_account_type_expenses","l10n_cn_small_business.l10n_chart_china_small_business","False"
-"l10n_cn_6603","Financial Expenses","6603","account.data_account_type_expenses","l10n_cn_small_business.l10n_chart_china_small_business","False"
-"l10n_cn_6701","Assets impairment Loss","6701","account.data_account_type_expenses","l10n_cn_small_business.l10n_chart_china_small_business","False"
-"l10n_cn_6711","Non-operating expenses","6711","account.data_account_type_expenses","l10n_cn_small_business.l10n_chart_china_small_business","False"
-"l10n_cn_6801","Income Tax Expense","6801","account.data_account_type_expenses","l10n_cn_small_business.l10n_chart_china_small_business","False"
-"l10n_cn_6901","Prior year income adjustment","6901","account.data_account_type_expenses","l10n_cn_small_business.l10n_chart_china_small_business","False"
+"id","code","name","reconcile","user_type_id/id","chart_template_id/id"
+"small_business2017_chart1001","1001","库存现金","False","account.data_account_type_liquidity","l10n_cn_small_business.l10n_chart_china_small_business"
+"small_business2017_chart1002","1002","银行存款","False","account.data_account_type_liquidity","l10n_cn_small_business.l10n_chart_china_small_business"
+"small_business2017_chart1012","1012","其他货币资金","False","account.data_account_type_current_assets","l10n_cn_small_business.l10n_chart_china_small_business"
```

Comment:
Why change xml id?  It is breaking stuff for breaking stuff.  If you kept l10n_cn_ and the same structure, we could really see the difference between the two.

Reply:
And sometimes you have a code of length 7, while you put code digits at 4

Reply:
@jco-odoo  code digit define the "minimal" length of account code, it keep adding trialling 00 to first level account code when it is "6". change it to "4" is the must we have from 10 years ago in openerp.

Yes, xml id should be changed back @Colinliz